spec/rspec/core/formatters/documentation_formatter_spec.rb

```diff
@@ -81,6 +81,36 @@ def execution_result(values)
 ")
     end
 
+    it "can output indented messages from within example group" do
+      root = RSpec.describe("root")
+      root.example("example") {|example| example.reporter.message("message")}
+
+      root.run(reporter)
+
+      expect(formatter_output.string).to eql("
+root
+  example
+    message
+")
+    end
+
+    it "can output indented messages" do
+      root = RSpec.describe("root")
+      context = root.describe("nested")
+      context.example("example") {}
+
+      root.run(reporter)
+
+      reporter.message("message")
+
+      expect(formatter_output.string).to eql("
+root
+  nested
+    example
+  message
```

Comment:
Quite surprise about this indent but this is probably because my mind doesn't clearly identify some spec converted to this output.

Reply:
What do you feel it should look like?

Reply:
It should be `0` indented, as this output isn't from a spec or group

Reply:
```
      root = RSpec.describe("root")
      context = root.describe("nested")
      reporter.message("message")
```

Is there some reason why "nested" should be indented but "message" should not?

Reply:
I'll update the behaviour as you requested but please think about whether it makes sense or not, personally, I feel that it should be indented the same way the title of a nested context is indented.